### PR TITLE
WeaveDemo on Minimesos! Local docker-machine or AWS.

### PR DESCRIPTION
**Pull request by philwinder**

### CODE REVIEW

**Comment by sashkachan** [comment]
When running `./minimesos-marathon install`, the script pulls the latest boot2docker with 1.12RC and minimesos is incompatible with it.
You can work around it by setting an env var, eg:

```
VIRTUALBOX_BOOT2DOCKER_URL=https://github.com/boot2docker/boot2docker/releases/download/v1.11.2/boot2docker.iso ./minimesos-marathon.sh install
```

File: deploy/minimesos-marathon/minimesos-marathon.sh [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+
+    while [[ $(status_code $MINIMESOS_MARATHON) != "302" ]] ; do
+        info "Wating for Marathon to respond ok"
+        sleep 1;
+    done
+
+    info "Starting services"
+    curl -q -XPOST -H 'Content-Type:application/json' -d @$MARATHON_FILE $MINIMESOS_MARATHON/v2/groups
+
+    if [ $(aws_instance) ] ; then
```
Comment (sashkachan) [inline deploy/minimesos-marathon/minimesos-marathon.sh]: This should be compared to return value 0.

**Comment by sashkachan** [comment]
👍

File: install/aws-minimesos/provision.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+sudo cp ~/.minimesos/bin/minimesos /usr/local/bin/minimesos
+
+# Install weave
+sudo curl -L git.io/weave -o /usr/local/bin/weave
+sudo chmod +x /usr/local/bin/weave
+
+# Clone repo to get deployment scripts
+git clone https://github.com/weaveworks/weaveDemo.git
+cd weaveDemo
+git checkout deploy/minimesos-marathon
```
Comment (philwinder) [inline install/aws-minimesos/provision.sh]: This checks out a branch. Replace with master when merging.

**Pull request merged**